### PR TITLE
Add test plan for QUARKUS-2483

### PR DESCRIPTION
**Pull request by michalvavrik**
Adds a test plan for [QUARKUS-2483](https://issues.redhat.com/browse/QUARKUS-2483).

### CODE REVIEW

File: QUARKUS-2483.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# QUARKUS-2483 - Kubernetes service binding support for Reactive SQL clients
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2483
+
+PR: https://github.com/quarkusio/quarkus/pull/25657
+
+In Quarkus 2.10 was added Kubernetes service binding support for following Reactive SQL Clients extensions:
```
Comment (rsvoboda) [inline QUARKUS-2483.md:7]: Pls also mention product support level, you can consult https://code.quarkus.redhat.com, DB2 TP is ditched with 2.13 - https://issues.redhat.com/browse/QUARKUS-2332

Basically just PostgreSQL and MySQL ones are supported, MSSQL as TP, the rest just on community level.

Comment (michalvavrik) [reply QUARKUS-2483.md]: Done, thanks.

File: QUARKUS-2483.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+In Quarkus 2.10 was added Kubernetes service binding support for 5 Reactive SQL Clients extensions.
+
+Clients with the [Production support](https://access.redhat.com/support/offerings/production/soc/):
+- Reactive PostgreSQL client
+- Reactive MySQL client
+
+Clients with the [Technology Preview features support](https://access.redhat.com/support/offerings/techpreview):
+- Reactive MS SQL client
+
+Client with the [Development support](https://access.redhat.com/support/offerings/developer/):
```
Comment (rsvoboda) [inline QUARKUS-2483.md:16]: Just community support, nothing from RH side

```suggestion
Clients with Quarkus community support:
```

Comment (michalvavrik) [reply QUARKUS-2483.md]: Done. I didn't know what `PREVIEW` label means, thank you.

**Review by mjurc — APPROVED** [review]
This looks good to me, thanks for the PR.

**Pull request merged**